src/pytorch_lightning/CHANGELOG.md

```diff
@@ -113,6 +113,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ### Changed
 
+- When training with `precision=16` on IPU, the cast has been moved off the IPU onto the host, making the copies from host to IPU cheaper ([#13880](https://github.com/Lightning-AI/lightning/pull/13880))
```

Comment:
Since we will be considering this change a bug-fix, please move this into the "Fixed" section